azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -530,46 +544,6 @@ private void configureFlowLevelMetrics(FlowRunner flowRunner) {
 
   }
 
-  private void setupFlow(ExecutableFlow flow) throws ExecutorManagerException {
```

Comment:
Thank you! Not only that. 
I thought of bringing the code for preparing flows at one place. So that I can update this easily.